### ISSUE TITLE
global.___INIT_METEOR_FAST_REFRESH is not a function when HMR is enabled and importing the module with a name "global"

### ISSUE THREAD

**Issue by dovydaskukalis** [meteor/meteor#11744]
Version: 2.5 on Mac OS 12.0.1

We have noticed that in Meteor 2.5 importing CKEditor crashes the app. Digging deeper we found that internally CKEditor has import like this `import global from './global';` which seems to break the generated app.js bundle because variable `global` is being overwritten by the imported one, which doesn't have `___INIT_METEOR_FAST_REFRESH` variable defined.

The generated contents of `app.js` looks like this. It sets `let global;` which makes subsequent ` global.___INIT_METEOR_FAST_REFRESH(module);` to crash.

```
!function (module1) {
  module1.export({
    FailingComponent: () => FailingComponent
  });
  let React;
  module1.link("react", {
    default(v) {
      React = v;
    }

  }, 0);
  let global;
  module1.link("./utils/global", {
    default(v) {
      global = v;
    }

  }, 1);

  global.___INIT_METEOR_FAST_REFRESH(module);

  const FailingComponent = () => {
    // Do something with global
    console.log(global.document);
    return /*#__PURE__*/React.createElement("div", null, "Failing component");
  };

  _c = FailingComponent;

  var _c;

  $RefreshReg$(_c, "FailingComponent");
}.call(this, module);
```

Reproduction: https://github.com/dovydaskukalis/meteor-hmr-issue

**Comment by SkariDamm** [comment]
Any news on a solution to this problem or a workaround?

**Comment by dovydaskukalis** [comment]
Workaround we use now is starting Meteor with `DISABLE_REACT_FAST_REFRESH=1` which is not great but at least it works.

**Comment by filipenevola** [comment]
This should be fixed in Meteor 2.5.2.
